tests/test_resolutions.py

```diff
@@ -166,15 +165,15 @@ def test_the_test(self) -> None:
 
     def test_description(self) -> None:
         test_entry = {"spam": "eggs"}
-        test_entries = {"tree": "larch", "spam": "eggs"}
+        test_entries = {"tree": 1234, "spam": "eggs"}
```

Comment:
1234 is not a tree!

Reply:
lol, I knew I should have changed the key name.  I wanted to show how different objects print when in a dictionary.
Even though it's not part of the test to verify how hamcrest is formatting integers, I thought it was important to at LEAST show it.

```
tests/test_resolutions.py::TestContainsTheEntry::test_the_test 
-------------------------------- live log call ---------------------------------
... hoping it's a mapping with the entry 'key'->'value'
    => a dictionary containing {'key': 'value'}
... hoping it's a mapping with the entries 'key1'->'value1', 'key2'->'value2'
    => a dictionary containing {'key1': 'value1', 'key2': 'value2'}
... hoping it's a mapping with the entry 'key2'->12345
    => a dictionary containing {'key2': <12345>}
... hoping it's a mapping with the entry 'key3'->'something3'
    => a dictionary containing {'key3': 'something3'}
PASSED                                                                   [ 70%]
```